build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java

```diff
@@ -107,11 +107,8 @@ public void storeResult(BuildTask buildTask, BuildResult buildResult) throws Dat
                 return;
             }
 
-            // Build driver results
-            buildRecordBuilder.endTime(Date.from(Instant.now()));
```

Comment:
The end time is now set in initBuildRecordBuilder method.  Reviewing this, it looks like setting the endTime should also be removed from storeResult with an error and storeRejected since they are now redundant.